frontend/src/core/services/auth.service.ts

```diff
@@ -0,0 +1,24 @@
+import { HttpResponse } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { HttpService } from './http.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+
+  public routePrefix = '/api/user';
+
+  constructor(
+    private httpService: HttpService
+  ) { }
+
+  loginUser(email: string, password: string):Observable<HttpResponse<{ assessToken: string }>> {
+    return this.httpService.postFullRequest(this.routePrefix + '/login', { email: email, password: password });
+  }
+
+  registerUser(email: string, password: string, name: string):Observable<HttpResponse<{ assessToken: string }>> {
```

Comment:
Same as above

Reply:
Also register does not return access token. Just empty OK. We'll have to redirect users to login page or `Check your email for confirmation link` page